front/app/containers/InitiativesShow/ReactionControl/Status/ProposedNotReacted.tsx

```diff
@@ -1,216 +1,31 @@
-  word-wrap: break-word;
-  word-break: break-word;
-
-  a,
-  button {
-    color: ${colors.teal};
-    font-size: ${fontSizes.base}px;
-    line-height: normal;
-    font-weight: 400;
-    text-align: left;
-    text-decoration: underline;
-    white-space: normal;
-    overflow-wrap: break-word;
-    word-wrap: break-word;
-    word-break: break-all;
-    word-break: break-word;
-    hyphens: auto;
-    display: inline;
-    padding: 0px;
-    margin: 0px;
-    cursor: pointer;
-    transition: all 100ms ease-out;
-
-    &:hover {
-      color: ${darken(0.15, colors.teal)};
-      text-decoration: underline;
-    }
-  }
-`;
-
-const disabledMessages: {
-  [key in InitiativePermissionsDisabledReason]: MessageDescriptor;
-} = {
-  not_permitted: messages.votingNotPermitted,
-  not_in_group: globalMessages.notInGroup,
-};
-
-const ProposedNotReacted = ({
-  onReaction,
-  initiative,
-  initiativeSettings,
-  disabledReason,
-  userReacted,
-  onCancelReaction,
-}: StatusComponentProps) => {
-  const theme = useTheme();
-
-  const thresholdReachedTooltip = (
-    <IconTooltip
-      icon="info-outline"
-      iconColor={theme.colors.tenantText}
-      theme="light"
-      placement="bottom"
-      content={
-        <T value={initiativeSettings.threshold_reached_message} supportHtml />
-      }
-    />
-  );
-
-  const tippyContent = disabledReason ? (
-    <TooltipContent id="tooltip-content" className="e2e-disabled-tooltip">
-      <TooltipContentIcon name="lock" ariaHidden />
-      <TooltipContentText>
-        <FormattedMessage {...disabledMessages[disabledReason]} />
-      </TooltipContentText>
-    </TooltipContent>
-  ) : null;
+import Status from './index2';
 
+const ProposedNotReacted = (props: StatusComponentProps) => {
   return (
-    <Container>
-      <Box ml="auto">
-        <CountDown targetTime={initiative.attributes.expires_at} />
-      </Box>
-      <StatusIcon ariaHidden name="bullseye" />
-      <StatusExplanation>
-        <OnDesktop>
-          <FormattedMessage
-            {...messages.proposedStatusExplanation}
-            values={{
-              votingThreshold: initiativeSettings.reacting_threshold,
-              proposedStatusExplanationBold: (
-                <b>
-                  <FormattedMessage
-                    {...messages.proposedStatusExplanationBold}
-                  />
-                </b>
-              ),
-            }}
-          />
-          {thresholdReachedTooltip}
-        </OnDesktop>
-        <OnMobile>
-          <FormattedMessage
-            {...messages.proposedStatusExplanationMobile}
-            values={{
-              daysLeft: getPeriodRemainingUntil(
-                initiative.attributes.expires_at
-              ),
-              votingThreshold: initiativeSettings.reacting_threshold,
-              proposedStatusExplanationMobileBold: (
-                <b>
-                  <FormattedMessage
-                    {...messages.proposedStatusExplanationMobileBold}
```

Comment:
Don't know why we had these differences. Simplifying to the same text on mobile & desktop.